handout/features/arrow_functions.md

```diff
@@ -1,7 +1,7 @@
 # Arrow Functions
 
-ES6 offers some new syntax for dealing with `this`: "arrow functions".  Arrow
-function also make working with "higher order" functions (functions that take
+ES6 offers some new syntax for dealing with `this`: "arrow functions".  
+Arrow functions also make working with "higher order" functions (functions that take
```

Comment:
'.. make working with ... much easier to work with'

Probably a place where our text could be refined even more :)